### DISCUSSION TITLE
Calling text_dataset_from_directory from keras.utils

### DISCUSSION THREAD

**Discussion by flaviu22** [serizba/cppflow#224] [General]
Is there possible to call **text_dataset_from_directory** from **tf.keras.utils** ?

Basically, I intend to use this code: [](https://www.tensorflow.org/tutorials/keras/text_classification) by **cppflow**, and the first keras methos is the one from above ... its possible ?

Thank you.

**Comment by serizba** [comment]
Cppflow ops wraps the ops in `tf.raw_ops`, and unfortunately `text_dataset_from_directory` is not defined there.
